wiki/osu!tourney/en.md

```diff
@@ -1,56 +1,69 @@
-**osu!tourney** is the designated multi-spectator client used to view and stream entire online multiplayer matches at once.
+# osu!tourney
 
-The access is limited to tournament hosts and their staff. Tournament hosts have to contact **tournaments@ppy.sh** to apply for access, which is limited for the time needed to conclude the tournament they host.
-Using the client is not meant for public right now and requires approval from the game administration.
+**osu!tourney** is the designated multi-spectator client used to view and stream entire online multiplayer matches at once.
```

Comment:
L3: Revise. Suggested ``**osu!tourney** is the official multi-spectator client used to view and stream entire multiplayer matches at once in osu!.``